types/event_record.go

```diff
@@ -81,7 +81,71 @@ type EventRecords struct {
+	Elections_NewTerm                  []EventElectionsNewTerm                  //nolint:stylecheck,golint
+	Elections_EmptyTerm                []EventElectionsEmptyTerm                //nolint:stylecheck,golint
+	Elections_MemberKicked             []EventElectionsMemberKicked             //nolint:stylecheck,golint
+	Elections_MemberRenounced          []EventElectionsMemberRenounced          //nolint:stylecheck,golint
+	Elections_VoterReported            []EventElectionsVoterReported            //nolint:stylecheck,golint
+	Identity_IdentitySet               []EventIdentitySet                       //nolint:stylecheck,golint
+	Identity_IdentityCleared           []EventIdentityCleared                   //nolint:stylecheck,golint
+	Identity_IdentityKilled            []EventIdentityKilled                    //nolint:stylecheck,golint
+	Identity_JudgementRequested        []EventIdentityJudgementRequested        //nolint:stylecheck,golint
+	Identity_JudgementUnrequested      []EventIdentityJudgementUnrequested      //nolint:stylecheck,golint
+	Identity_JudgementGiven            []EventIdentityJudgementGiven            //nolint:stylecheck,golint
+	Identity_RegistrarAdded            []EventIdentityRegistrarAdded            //nolint:stylecheck,golint
+	Recovery_RecoveryCreated           []EventRecoveryCreated                   //nolint:stylecheck,golint
+	Recovery_RecoveryInitiated         []EventRecoveryInitiated                 //nolint:stylecheck,golint
+	Recovery_RecoveryVouched           []EventRecoveryVouched                   //nolint:stylecheck,golint
+	Recovery_RecoveryClosed            []EventRecoveryClosed                    //nolint:stylecheck,golint
+	Recovery_AccountRecovered          []EventRecoveryAccountRecovered          //nolint:stylecheck,golint
+	Recovery_RecoveryRemoved           []EventRecoveryRemoved                   //nolint:stylecheck,golint
+	Sudo_Sudid                         []EventSudoSudid                         //nolint:stylecheck,golint
+	Sudo_KeyChanged                    []EventSudoKeyChanged                    //nolint:stylecheck,golint
+	Sudo_SudoAsDone                    []EventSudoAsDone                        //nolint:stylecheck,golint
+	Council_Proposed                   []EventCollectiveProposed                //nolint:stylecheck,golint
+	Council_Voted                      []EventCollectiveProposed                //nolint:stylecheck,golint
+	Council_Approved                   []EventCollectiveApproved                //nolint:stylecheck,golint
+	Council_Disapproved                []EventCollectiveDisapproved             //nolint:stylecheck,golint
+	Council_Executed                   []EventCollectiveExecuted                //nolint:stylecheck,golint
+	Council_MemberExecuted             []EventCollectiveMemberExecuted          //nolint:stylecheck,golint
+	Council_Closed                     []EventCollectiveClosed                  //nolint:stylecheck,golint
```

Comment:
Are you sure the module is named `Council`? Events are matched by key, if the module was called `Collective`, this would not work.

Reply:
> Also, considering how substrate lets you have different module name, It is the client responsibility to add those events under the new module name.

Hence this comment :-D

Reply:
If we go with this, any module can have a name :-D

Reply:
Actually, I want to remove this from the Records. Every client should give them how it come in